spec/requests/v1/items_request_spec.rb

```diff
@@ -38,20 +38,41 @@
 
         items = JSON.parse(response.body, symbolize_names: true)
 
+        
+
         items[:data].each do |item|
-            expect(item[:id].to_i).to be_an(Integer)
+            expect(item[:id]).to be_an(String)
+            # check for type
 
-            expect(item[:attributes]).to have_key(:name)
             expect(item[:attributes][:name]).to be_a(String)
 
-            expect(item[:attributes]).to have_key(:description)
             expect(item[:attributes][:description]).to be_a(String)
 
-            expect(item[:attributes]).to have_key(:unit_price)
             expect(item[:attributes][:unit_price]).to be_a(Float)
         end
     end
 
+    it 'displays all items sorted by price' do
+        get '/api/v1/items?sort=unit_price'
+
+        expect(response).to be_successful
+
+        items = JSON.parse(response.body, symbolize_names: true)
+
+        prices = items[:data].map { |item| item[:attributes][:unit_price] }
+        expect(prices).to eq(prices.sort)
+
+        items[:data].each do |item|
```

Comment:
I am curious about this part of the test? Is there an expectation to check this on every test? This it block is testing that the items are sorted when the query param is in place. This is checked in line 63, but this bit down here is testing for something else. Its not bad to have thorough testing but i'm curious about what the justification is in this context.